### PR TITLE
[REMOTE-WDPT-0602] - Pedro Lamy

### PR DESCRIPTION
**Pull request by PedroMMLamy**
Missing iterations 5 and 6.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,6 +22,25 @@ mongoose
...
     // Run your code here, after you have insured that the connection was made
   })
+
+  Recipe.create({ title: "Feijoada", level: "UltraPro Chef", ingredients: ["rice", "chouriço", "morcela", "farinheira", "pork meat", "Masa"], cuisine: "Portugal", dishType: "dinner", image: "https://honest-food.net/wp-content/uploads/2013/12/portuguese-feijoada-320x320.jpg", duration: 120, creator: "Pedro" })
+  .then((recipe) => console.log(`Saved Recipe ${recipe}`))
+
+  .then(() => Recipe.insertMany(data))
+      .then((dataRecipe) =>
+        dataRecipe.forEach((recipes) => {
+          console.log("Create Recipes:", recipes.title);
```
Comment (valmgisbert) [inline index.js:32]: You should practice template literals `${}` for your console.logs/return strings, to become more comfortable with it

File: models/Recipe.model.js [hunk 2]
```diff
@@ -3,8 +3,16 @@ const Schema = mongoose.Schema;
...
+  image: {type: String, default: 'https://honest-food.net/wp-content/uploads/2013/12/portuguese-feijoada-320x320.jpg'},
+  duration: {type: Number, min: 0},
+  creator: {type: String},
+  created: {type: Date, default: new Date()}
 
-const Recipe = mongoose.model('Recipe', recipeSchema);
+});
 
-module.exports = Recipe;
+const Recipe = mongoose.model('Recipe', recipeSchema);
```
Comment (valmgisbert) [inline models/Recipe.model.js:18]: remember to export your module; a way to do it in one sentence could be: module.exports = mongoose.model('Recipe', recipeSchema);

**Comment by valmgisbert** [comment]
Remember that if you're going to chain multiple promises, you should use Promise.all(). My recommendation is for you to structure your CRUD methods in blocks at first to understand every single one of them and then refactor it to chained. Also, work on your spacing for an easier to read code. If you're using template literals for one, you should stick to it for the rest of your returning strings/console.logs to maintain consistency.

**Comment by PedroMMLamy** [comment]
Val thank you for your helpful feedback. I will work on it

A sábado, 1/08/2020, 11:40, valmgisbert <notifications@github.com> escreveu:

**Pull request closed without merging**